multiapps-common/src/main/java/org/cloudfoundry/multiapps/common/util/XmlUtil.java

```diff
@@ -27,6 +12,16 @@
 import org.xml.sax.SAXException;
 import org.xml.sax.SAXParseException;
 
+import javax.xml.XMLConstants;
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.parsers.ParserConfigurationException;
+import javax.xml.validation.SchemaFactory;
+import java.io.*;
```

Comment:
try not to use wildcards

Reply:
Fixed. Seems to have been caused by a configuration in the IDE